MAINTAINERS.yml

```diff
@@ -3625,8 +3625,8 @@ Silabs Platforms:
     - dts/arm/silabs/
     - dts/bindings/*/silabs*
     - drivers/*/*gecko*
-    - drivers/bluetooth/hci/hci_silabs_*
```

Comment:
Since the below glob pattern for silabs isn't recursive, is the hci_silabs_efr32.c file still matched when this is removed?

Reply:
ya should probably be `drivers/**/*silabs*`?

Reply:
Indeed. I now use `drivers/**/*silabs*`, `drivers/**/*siwx91x*` and `drivers/**/*gecko*`.